db/migrate/20190712135032_set_provisioned_state_to_services.rb

```diff
@@ -0,0 +1,12 @@
+class SetProvisionedStateToServices < ActiveRecord::Migration[5.1]
+  class Service < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    say_with_time("Update service lifecycle state values") do
+      Service.in_my_region.where(:lifecycle_state => nil, :retired => nil).update(:lifecycle_state => 'provisioned')
```

Comment:
Lifecycle_state can be either 'provisioned' or 'error_in_provisioning' depending on the request status.
Is there any issue if we blindly set all lifecycle_state to 'provisioned'?

Reply:
Tina answered this in a chat message but it was documented [here](https://github.com/ManageIQ/manageiq/pull/18943#discussion_r303019962).

Reply:
I think `update_all` would be better here as it will not instantiate all of the records by sending a single query.